### PR TITLE
autofill telemetry actions!

### PR DESCRIPTION
**Pull request by sashei**
Fixes only getting telemetry for some autofill actions!

### CODE REVIEW

File: app/src/main/java/mozilla/lockbox/action/AutofillAction.kt [hunk 1]
```diff
@@ -7,12 +7,14 @@
...
-sealed class AutofillAction : Action {
-    data class Complete(val login: ServerPassword) : AutofillAction()
-    data class CompleteMultiple(val logins: List<ServerPassword>) : AutofillAction()
-    data class Error(val error: Throwable) : AutofillAction()
-    object Authenticate : AutofillAction()
-    object Cancel : AutofillAction()
+sealed class AutofillAction(
+    override val eventMethod: TelemetryEventMethod,
+    override val eventObject: TelemetryEventObject
+) : TelemetryAction {
```
Comment (eliserichards) [inline app/src/main/java/mozilla/lockbox/action/AutofillAction.kt]: 👏

**Comment by devinreams** [comment]
@sashei is it easy enough to generate some quick events and @irrationalagent for you to check the pipeline before this gets merged? just to make _super_ sure? I'm happy to merge as-is too and see...

**Comment by sashei** [comment]
@irrationalagent @devinreams sure, I just went thru and did some autofilling and then returned the app to make sure it got uploaded, lmk how that looks for you!!

**Comment by devinreams** [comment]
Confirmed new events are good, thanks @irrationalagent @sashei. Merging! 🚀

**Pull request merged**